### ISSUE TITLE
How to change the starting level of the charging animation?

### ISSUE THREAD

**Issue by mmadera2007** [Umerbhat/react-battery-gauge#10]
Can't seem to figure out how to control the starting level of the charging animation.  I would like to be able to change this to sort of indicate the level of the charge of the battery.

**Comment by Umerbhat** [comment]
Thanks for raising your query, right now start value for animation is kept 1% by default.

```
  const chargingValue = useCounterAnimation({
    startValue: 1,
    value: maxValue,
    enabled: charging,
    iterationCount: 'infinite',
    duration: 2000,
  });

```

We will make it customisable in coming weekend and release new version. Mean while, feel free to raise pull request.

**Comment by mmadera2007** [comment]
Thanks for your prompt reply.  Though I'm a bit new to React, I gave it a shot and created this PR: https://github.com/Umerbhat/react-battery-gauge/pull/11.  I added a new example to your examples page as well.

**Comment by Umerbhat** [comment]
Fixed in `v1.0.2`
